setup.py

```diff
@@ -5,18 +5,25 @@
     long_description = fh.read()
 
 setuptools.setup(
-    name="pello",
+    name="Pello",
     version="1.0.0",
-    author="",
-    author_email="",
-    description="A package that makes it easy to create Pypi packages",
+    author="Tomáš Hrnčiar",
+    author_email="tomas.hrnciar@me.com",
+    description="An example Python Hello World package",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="",
+    url="https://github.com/hrnciar/pello",
+    license="Public Domain",
     packages=setuptools.find_packages(),
     classifiers=[
         "Programming Language :: Python :: 3",
         "License :: OSI Approved :: MIT License",
```

Comment:
CC0

Reply:
Still marked as MIT

Reply:
Sorry, I thought this comment was duplicate with the one above because I didn't notice that you were pointing at classifier. It's updated now.